pkgs/development/tools/misc/unixbench/default.nix

```diff
@@ -0,0 +1,102 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchpatch
+, makeWrapper
+, pandoc
+, installShellFiles
+, perl
+, xorg
+, libGL
+, bash
+, coreutils
+, unixtools
+, targetPackages
+, gnugrep
+, gawk
+, withGL? true
+, withX11perf? true
+}:
+
+stdenv.mkDerivation rec {
+  pname = "unixbench";
+  version = "unstable-2023-02-27";
+
+  src = fetchFromGitHub {
+    owner = "kdlucas";
+    repo = "byte-unixbench";
+    rev = "81e9de58c50566da8ae45c0ea7a55e7a7066ea75";
+    hash = "sha256-mAgcwvfYDrPYqrjTzlRBYlLo7DK93CPi2iavB5+oagk=";
+  };
+
+  patches = [
+    ./common.patch
+  ];
+
+  patchFlags = [ "-p2" ];
+
+  sourceRoot = "source/UnixBench";
+
+  nativeBuildInputs = [
+    makeWrapper
+    pandoc
+    installShellFiles
+  ];
+
+  buildInputs = [ perl ] ++ lib.optionals withGL [
+    xorg.libX11
+    xorg.libXext
+    libGL
+  ];
+
+  runtimeDependencies = [
+    coreutils
+    unixtools.nettools
+    unixtools.locale
+    targetPackages.stdenv.cc
+    gnugrep
+    gawk
+  ] ++ lib.optionals withX11perf [
+    xorg.x11perf
+  ];
+
+  makeFlags = lib.optionals withGL [ "GRAPHIC_TESTS=defined" ];
```

Comment:
```suggestion
  makeFlags = [ "CC=${stdenv.cc.targetPrefix}cc" ] ++ lib.optionals withGL [ "GRAPHIC_TESTS=defined" ];
```

Reply:
Is this by default in the cross stdenv?

Reply:
No. There have probably been packages that use `CC` as something else, like `openjdk`.